server/api/src/main/java/org/hawkular/apm/server/api/utils/ProducerInfoCacheUtil.java

```diff
@@ -202,6 +202,7 @@ public void initialiseFromSpans(String tenantId, List<Span> items) throws RetryA
                 pi.setDuration(span.getDuration());
                 pi.setTimestamp(span.getTimestamp() / 1000);
                 pi.setFragmentId(span.getId());
+                pi.getProperties().addAll(span.properties());
 
                 // TODO: SET IP ADDRESS
```

Comment:
Should we also populate the IP address field of derived information using the Endpoint ipv4?